bfdd/bfd.c

```diff
@@ -387,17 +387,13 @@ struct bfd_session *ptm_bfd_sess_find(struct bfd_pkt *cp,
 
 	/* Search for session without using discriminator. */
 	ifp = if_lookup_by_index(ifindex, vrfid);
-	if (vrfid == VRF_DEFAULT) {
-		/*
```

Comment:
does that mean that later, if the show bfd peer command relies on that command, then 'show bfd peer' will display only sessions from default vrf, and not all sessions from all vrfs?
I mean this change of behaviour should be explicitly notified in the commit log